packages/block-library/src/navigation/edit.js

```diff
@@ -199,6 +199,7 @@ export default compose( [
 			parent: 0,
 			order: 'asc',
 			orderby: 'id',
+			context: 'view',
```

Comment:
We can't just switch to using the "view" context because it means we're having inconsistent entities format in the state depending on the caller. I think right now, we just can't use `getEntityRecords`. Instead, we should use `apiFetch` in these components directly. One way to make this easier and avoid repeating the same lifecycle/state handling for all apiFetch usage is to consider a `useApiFetch` hook.

```
const { data, status, error } = apiFetch( { path: 'url' } )
```

Reply:
@youknowriad So using `apiFetch` directly and not updating the "Redux" state at all, only local component state?

Also even if we do use a more direct method to make the API request, the data will still be wrong unless we switch the context to `view`. Is this what you are recommending?

Reply:
yes, local state + view context.

And API fetch is just a simple wrapper around wp.fetch to make it work in WP context https://developer.wordpress.org/block-editor/packages/packages-api-fetch/

Reply:
Looks like Latest Posts uses it https://github.com/WordPress/gutenberg/blob/d0e3c26ec295924d96a658640e2cc6fb118b2eca/packages/block-library/src/latest-posts/edit.js#L49